### PR TITLE
Updating in light of tutorial comments

### PR DESCRIPTION
**Pull request by ionFreeman**
Replacing deprecated Pair with Tuple2 and concatenation with string interpolation

### CODE REVIEW

File: tutorials/tour/sequence-comprehensions.md [hunk 1]
```diff
@@ -32,10 +32,10 @@ Here is a more complicated example which computes all pairs of numbers between `
       def foo(n: Int, v: Int) =
         for (i <- 0 until n;
              j <- i until n if i + j == v) yield
-          Pair(i, j);
+          Tuple2(i, j);
       foo(20, 32) foreach {
         case (i, j) =>
-          println("(" + i + ", " + j + ")")
+          println(s"($i, j)")
```
Comment (SethTisue) [inline tutorials/tour/sequence-comprehensions.md]: missing dollar sign on j

File: tutorials/tour/sequence-comprehensions.md [hunk 2]
```diff
@@ -32,10 +32,10 @@ Here is a more complicated example which computes all pairs of numbers between `
       def foo(n: Int, v: Int) =
         for (i <- 0 until n;
              j <- i until n if i + j == v) yield
-          Pair(i, j);
+          Tuple2(i, j);
```
Comment (SethTisue) [inline tutorials/tour/sequence-comprehensions.md]: it's more idiomatic to simply write `(i, j)` by itself

**Comment by ionFreeman** [comment]
@SethTisue OK! Updated.

**Comment by SethTisue** [comment]
LGTM — thanks!

**Comment by ionFreeman** [comment]
Woo hoo! I’m a committer.

**Pull request merged**